src/born_digital_docs_scripts/make_sc.py

```diff
@@ -0,0 +1,147 @@
+
+        #below for testing purposes
+        # if len(str(path)) / 2 == 0: # just to create variation for now
+        #     interlacing.append([path, 'interlaced'])
+        # else:
+        #     interlacing.append([path, 'progressive'])
+    # print(interlacing)
+    return interlacing
+
+
+# Function take list of of [path, interlaced], create ffmpeg commands (need to adjust interlacing and service file path per command) (may need to create servicecopy folder before ffmpeg runs), return list of commands
+
+
+# Overwrite servicecopies, may need to add flag to ffmpeg to do this (-y?)
+def make_commands(files: list[list[Path,str]]) -> list[list[str]]:
+    commands = []
+
+    for x in files:
+        em_path = x[0]
+        em_path_str = str(em_path)
+        # PosixPath('test_ems/dir_3/data/EditMaster/sample_dig_3_em.mov')
+        base = em_path.parent.parent
+        '''
+        May be useful to include a check for service copy directory already existing. 
+        added because ffmpeg did not like the directory not already existing
+        '''
+        dest = base / 'ServiceCopies'
+        subprocess.run(['mkdir', f'{dest}'])
+        sc_path_str =  str(base / 'ServiceCopies' / em_path.name.replace("em.mov", "sc.mp4"))
+        if x[1] == 'interlaced':
+            cmd = ['ffmpeg', '-i', em_path_str, '-map', '0:v', '-map', '0:a', '-c:v', 'libx264', '-movflags', '+faststart', '-crf', '20', '-maxrate', '7.5M', '-bufsize', '7.5M', '-vf', 'yadif', '-c:a', 'aac', '-b:a', '320000', '-ar', '48000', sc_path_str]
+        else:
+            cmd = ['ffmpeg', '-i', em_path_str, '-map', '0:v', '-map', '0:a', '-c:v', 'libx264', '-movflags', '+faststart', '-crf', '20', '-maxrate', '7.5M', '-bufsize', '7.5M', '-c:a', 'aac', '-b:a', '320000', '-ar', '48000', sc_path_str]
+
+        commands.append(cmd)
+
+    return commands
+
+
+
+# # Function take list of commands, run each command, return list of sc files
+def make_sc(commands: list[list[str]]) -> list[str]:
+    sc = []
+    for c in commands:
+        # logging.DEBUG(f"Running this command {c}")
+        subprocess.run(c)
+        sc.append(c[-1])
+        logging.info(f"{c[-1]} created")
+ 
+    return sc
+
+# # Function take list of sc files, make rclone command, return list of commands
+def make_rclone(files: list[str], dest) -> list[list[str]]:
+    commands = []
+    
+    for sc in files:
+        fn = Path(sc).name
+        print(fn)
+        rc = ['rclone', 'copyto', sc, f'{dest}/{fn}', '-P']
+        commands.append(rc)
+
+    # print(commands)
+    return commands
+        
+# # Function take list of rclone commands, run each, return none
+def run_rclone(commands: list[list[str]]) -> None:
+    for c in commands:
+        print(c)
+        logging.info(f"transferring {c[2]}")
+        subprocess.run(c)
+        logging.info(f"{c[2]} has been transferred")
+
+    return None
+
+def main():
+    source = parse_args().source
+    dest = parse_args().dest
```

Comment:
you only needs to call parse_args once. It will return an object that has all of the parsed arguments.

idiomatically, I see this like this
```
args = parse_args()
get_em(args.source)
```